src/applications/gi/components/search/InstitutionSearchForm.jsx

```diff
@@ -47,9 +47,13 @@ function InstitutionSearchForm({
   }
 
   const header = gibctFilterEnhancement ? 'Refine search' : 'Keywords';
-  const keywordSearchLabel = gibctFilterEnhancement
+  let keywordSearchLabel = gibctFilterEnhancement
     ? 'Enter a school, location, or employer name'
```

Comment:
Should this be the line that is updated?

Reply:
For now, yes. This is one of those situations where we have a prod flag within a feature flag.

Reply:
I just liked using a separate condition to make it easy when we remove the prod flag.